### PR TITLE
Fix_line_break_ods

### PR DESCRIPTION
**Pull request by jefleponot**
FIX libreOffice multi line cells #108

### CODE REVIEW

**Comment by SheetJSDev** [comment]
Code-wise looks fine (not exactly rocket science), can you also update the `bits/81_writeods.js` file (https://github.com/SheetJS/sheetjs/blob/master/bits/81_writeods.js#L27) ?

File: xlsx.js [hunk 1]
```diff
@@ -20052,7 +20052,7 @@ var write_content_ods = (function() {
 		return escapexml(text)
 			.replace(/  +/g, function($$){return '<text:s text:c="'+$$.length+'"/>';})
 			.replace(/\t/g, "<text:tab/>")
-			.replace(/\n/g, "<text:line-break/>")
+			.replace(/\n/g, "</text:p><text:p>")
```
Comment (garrettluu) [inline xlsx.js:20055]: Curious as to why line breaks don't work in this context?

**Comment by nguyencongphu1998** [comment]
Sorry, I still meet this problem.

**Comment by reviewher** [comment]
b03ad9d40c0a94c3124f57e4c31084c3e98ee707

**Pull request closed without merging**